EMBRACE_Client/LoginViewController.h

```diff
@@ -11,6 +11,8 @@
 @interface LoginViewController : UIViewController {
     IBOutlet UITextField *firstNameField;
```

Comment:
Rename these variables because they don't describe the information we're actually collecting: School Code, Participant Code, Study Day, and Experimenter Name.